backend/app/main/survey_views.py

```diff
@@ -86,7 +86,7 @@ def redirect_to_first_question(response, survey):
 
 # Send a welcome message to the user
 def welcome_user(send_function, is_prev_response=False):
-    if not is_prev_response:
+    if is_prev_response:
```

Comment:
Maybe change the boolean variable's name to `no_prev_response`? Or change the code `(retrieve_prev_record(phone_number)=="NONE")` to match the boolean variable's meaning.

Reply:
I merged from master and changed the variable name!